cluster/addons/dns/kubedns-rc.yaml.in

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
+kind: ReplicationController
```

Comment:
isn't this basically the same as cluster/saltbase/salt/kube-dns/kubedns-rc.yaml.in ?  Why do we need both?  Can we just kill off cluster/addons/dns and leave a pointer to the new location?